app/views/objects/_sidebar.html.slim

```diff
@@ -15,17 +15,17 @@ div class="hidden lg:block w-1/4"
                 = render "objects/sidebar/section/link",
                     title: included_module.constant,
                     href: object_path(object: included_module.path)
-      - unless @object.ruby_methods.select(&:class_method?).empty?
+      - unless @object.ruby_class_methods.empty?
         = render "objects/sidebar/section", title: 'Class Methods' do
           ul class="font-mono text-sm"
-            - @object.ruby_methods.select(&:class_method?).sort_by(&:name).each do |m|
+            - @object.ruby_class_methods.sort_by(&:name).each do |m|
```

Comment:
I think we should also move the `sort_by(&:name)` into the `ruby_instance_methods` and `ruby_class_methods` also.

Reply:
I like this because then they're memoized sorted (instead of doing that twice or more per page), and realistically I can't see anyone wanting them in any other order.

Reply:
> I think we should also move the `sort_by(&:name)` into the `ruby_instance_methods` and `ruby_class_methods` also.

Honestly, it's the next to-do in my list for Ruby API. I've done this, but I'm confused: sometimes you write:

> split this into multiple PRs
> ...
> Multiple small PRs means maybe he can review a small PR per day and get everything out, instead of struggling to get through one large PR.

Then you write "let's do sort caching in split PR!".

I ask you (all) to be consistent, please.

Reply:
@AlexWayfer Since you created the `ruby_class_methods` and `.ruby_instance_methods` methods in this PR, I figured it'd make sense to also move the `sort_by` in as well (you're already changing all of the lines involved).

I think I confused everything because early on I missed how many files were definitely related to this PR. I thought it could be split into a much smaller PR, but when you did that, I realized that this PR really wasn't much smaller. That's my bad. Sorry. 🙁 I was just trying to make things easier, but I think I made them harder.